modin/core/execution/ray/implementations/pandas_on_ray/partitioning/partition.py

```diff
@@ -66,8 +66,7 @@ def get(self):
         pandas.DataFrame
             The object from the Plasma store.
         """
-        if len(self.call_queue):
```

Comment:
There is no need to check the length, this is done in `drain_call_queue` function itself.